src/main/java/com/patsi/controller/SocialMediaUserController.java

```diff
@@ -0,0 +1,92 @@
+package com.patsi.controller;
+
+
+import com.patsi.bean.SocialMediaUser;
+import com.patsi.enums.AccountStatus;
+import com.patsi.service.SocialMediaService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.*;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.client.RestTemplate;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@Validated
+@RequestMapping("/socialMedia")
+@CrossOrigin
+public class SocialMediaUserController {
+    Logger log = LoggerFactory.getLogger(RecipeController.class);
+
+    @Autowired
+    private SocialMediaService socialMediaService;
+
+    public String getUidFromToken ( String token ) {
```

Comment:
Think this method should be extracted to a separated service. Let say UserProfile Service?

Reply:
Also, remove those extra spacing, should be:
`public String getUidFromToken(String token) {`

Reply:
fixed